cdap-runtime-spi/src/main/java/io/cdap/cdap/runtime/spi/launcher/JobDetails.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.runtime.spi.launcher;
+
+import java.util.Objects;
+
+/**
+ * Represents launched job details.
+ */
+public class JobDetails {
+  private final String status;
```

Comment:
will we be adding more things to this class?

Reply:
should this be an enum rather than a string?

Reply:
> will we be adding more things to this class?
we may add more things depending on what other information is needed by monitoring piece. In general, having this has a class is good because this would provide flexibility to add to it in future.


> should this be an enum rather than a string?
yea its good to have it as enum.